From 01f43c21023184ae3a2cf9b19f9183c8a4b5d251 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Mon, 20 Aug 2018 09:17:58 -0700
Subject: [PATCH 2/2] dgit-maint-debrebase(7): Add crossreference to "ILLEGAL
 OPERATIONS"

There are three items under "ILLEGAL OPERATIONS":

1. We already refer to "OTHER MERGES".

2. This commit adds a crossreference to the second item.

3. The information under "Editing debian/patches" is already basically
   included in dgit-maint-debrebase(7).

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index b8f2640..16b65b3 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -661,6 +661,7 @@ git-rebase without a base argument will often
 start the rebase too early,
 and should be avoided.
 Run git-debrebase instead.
+See also "ILLEGAL OPERATIONS" in git-debrebase(5).
 
 =head1 SEE ALSO
 
-- 
2.11.0

